packages/components/src/local/molecules/json-editor/index.tsx

```diff
@@ -128,12 +131,15 @@ export const JsonEditor: React.FC<Props> = ({
     document.addEventListener('keydown', handleKeyDown)
 
     if (nextEditor) {
-      const messageJson = expiredStorage.getItem(genLocalStorageId())
+      // only retrieve from expired content if we haven't been provided with message content
```

Comment:
@lilitkarapetyan - in the code below, I only try to retrieve the message from storage if there isn't a `messageContent`.  Does that look ok to you?

Reply:
Hello Ian, I think it's good.